Provide hooks to permit compilation in mock, where multipls gids do not exist
--- admin/daemontools-0.76/src/Makefile.exit	2001-07-12 12:49:49.000000000 -0400
+++ admin/daemontools-0.76/src/Makefile	2022-12-31 18:59:39.157137685 -0500
@@ -165,7 +165,8 @@
 
 hasshsgr.h: chkshsgr choose compile hasshsgr.h1 hasshsgr.h2 load \
 tryshsgr.c warn-shsgr
-	./chkshsgr || ( cat warn-shsgr; exit 1 )
+	#./chkshsgr || ( cat warn-shsgr; exit 1 )
+	./chkshsgr || cat warn-shsgr
 	./choose clr tryshsgr hasshsgr.h1 hasshsgr.h2 > hasshsgr.h
 
 haswaitp.h: choose compile haswaitp.h1 haswaitp.h2 load trywaitp.c
